CommonCompilerConfig.cmake

```diff
@@ -245,17 +248,17 @@ else()
 	#
 	# Determine stdlib settings
 	#
-	set(stdlib_cxx "")
-	set(stdlib_common "")
+	set(stdlib_cxx)
+	set(stdlib_common)
 
 	if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-	  set(stdlib_cxx "${stdlib_cxx} -stdlib=libc++")
+	  set(stdlib_cxx ${stdlib_cxx} -stdlib=libc++)
```

Comment:
interesting, why it didn't work as string? This is a common way to merge $PATHs

Reply:
CMake has many ways to set the compile flags.  I wanted to use [target_compile_options()](https://cmake.org/cmake/help/v3.0/command/target_compile_options.html) method because it is target specific and it expanded the generator expressions (needed for Release and Debug on Windows).  This uses a list rather than a string as input but no place does it tell you that in the documentation.  If you use the quotes in the Set() you get a string otherwise it creates a semicolon separated list.  arg1;arg2;arg3